src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-card-pop-up/ubs-admin-tariffs-card-pop-up.component.ts

```diff
@@ -160,7 +161,7 @@ export class UbsAdminTariffsCardPopUpComponent implements OnInit, OnDestroy {
       .pipe(takeUntil(this.unsubscribe))
       .subscribe((res: Couriers[]) => {
         this.couriers = res;
-        this.couriersName = this.couriers.map((item) => item.nameUk);
+        this.couriersName = this.couriers.map((item) => (this.currentLanguage === Language.UA ? item.nameUk : item.nameEn));
```

Comment:
Replace `this.currentLanguage === Language.UA ? item.nameUk : item.nameEn` with function 'getLangValue()' from Language service